### ISSUE TITLE
NTR: Skin: burrows with thin serpiginous lines (Skin and mucosa)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8017]
New term request Skin: burrows with thin serpiginous lines (seidel_190411134030):   UMLS: 0423769

**Comment by pnrobinson** [comment]
@MickeySegal
is this intended for scabies?

**Comment by MickeySegal** [comment]
Yes.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000960
- label: Serpiginous burrow
- parents: [Localized skin lesion (HP:0011355)](https://hpo.jax.org/app/browse/term/HP:0011355)
- pmid: PMID:34615594
- def: A slightly raised and often reddish serpentine or curvilinear path in the skin.
- comment: Serpiginous refers to the wavy path of the burrows that is said to resemble the tracks of a snake. These lesions are characteristic of scabies and some other infections.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8017
- synonyms:
